scripts/zkai_native_attention_mlp_rmsnorm_label_policy_gate.py

```
@@ -0,0 +1,693 @@
+    if frontier.get("frontier_win_claimed") is not False:
+        raise LabelPolicyError("frontier overclaim")
+    if frontier.get("nanozk_win_claimed") is not False:
+        raise LabelPolicyError("NANOZK overclaim")
+    if frontier.get("nanozk_workload_matched") is not False:
+        raise LabelPolicyError("NANOZK workload drift")
+    if frontier != expected_frontier:
+        raise LabelPolicyError("frontier body drift")
+
+    inventory = _list(payload.get("label_inventory"), "label inventory")
+    if tuple(item.get("name") for item in inventory if isinstance(item, dict)) != LABEL_INVENTORY:
+        raise LabelPolicyError("label inventory order drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)